### ISSUE TITLE
report invalid references from `@computedFrom`

### ISSUE THREAD

**Issue by atsu85** [MeirionHughes/aurelia-template-lint#128]
Following

``` TypeScript
      export class Foo{
        field1: number;
        field2: number;

        @computedFrom("field1", "missingField1", "field2", "missingField2")
        get computedField(){
          return this.field1 + this.field2;
        }
      }
```

``` HTML
      <template>
        \${computedField}
      </template>
```

should report that missingField1 and missingField2 can't be found in type Foo

**Comment by atsu85** [comment]
I'll send a PR with test soon

**Comment by atsu85** [comment]
The [PR](https://github.com/MeirionHughes/aurelia-template-lint/pull/129) is ready to be merged - it currently works with simple expressions (such as `@computedFrom("field1", "missingField1")`), but ignores expressions containing `.`, such as `@computedFrom("item.field")` - I think it could be left to another PR

**Comment by MeirionHughes** [comment]
okay, that is good enough; I'll mark it for revisit at a later date. thx,

**Comment by atsu85** [comment]
@MeirionHughes, I just started investigating how to check "item.field" - any hints? I probably need to create `AccessMember` expression out of string `"item.field"`, as it is done [here](https://github.com/MeirionHughes/aurelia-template-lint/blob/develop/source/ast.ts#L65), or how would You approach it?

**Comment by MeirionHughes** [comment]
You're probably be better off passing it through the attribute one instead; the next node would require interpolation. (you could wrap the string with `${_expression_}`) either way is a little hacky.

**Comment by MeirionHughes** [comment]
Disabled this as of 0.9.18. will reimplement in reworked code. 

This line https://github.com/MeirionHughes/aurelia-template-lint/blob/develop/source/rules/binding.ts#L504 results in errors if the property is part of a class that is imported by the view-model. 

In reworked code its easier to break this up into two processes: firstly to check that that view->viewmodel link is valid, in html file checks; and secondly to check the `@computedFrom` decorators, when processing source files.
